less/components/favorite.less

```diff
@@ -32,7 +32,11 @@
+  }
```

Comment:
Where are these data images coming from? I think LESS has some functions for generating this into the CSS from an image file. Might be easier to maintain.